client/src/components/ChatBox.tsx

```diff
@@ -1,16 +1,16 @@
-import React from "react";
+import React from 'react';
 //@ts-ignore
-import { ActionCableConsumer } from "react-actioncable-provider";
-import { API_ROOT, HEADERS } from "../constants";
+import { ActionCableConsumer } from 'react-actioncable-provider';
+import { API_ROOT, HEADERS } from '../constants';
 import {
   resourceToConversation,
   MessageResource,
   resourceToMessage
-} from "../lib/jsonApi";
-import { RailsRequest, Conversation } from "../types";
-import uuidv1 from "uuid/v1";
-import ChatMessages from "./ChatMessages";
-import NewMessageForm from "./NewMessageForm";
+} from '../lib/jsonApi';
```

Comment:
Could we figure out why this change is happening?

Reply:
I think I have a tool that automatically makes double quotes into single quotes (for strings). It was recommended during a tutorial

Reply:
Could we deactivate it? We should keep this consistent and prettier already changes everything to double quotes

Reply:
Deactivated. And everything seemed to revert back